corehq/apps/es/management/commands/elastic_sync_multiplexed.py

```diff
@@ -294,7 +294,7 @@ def remove_residual_indices(self):
         """
         existing_indices = es_manager.indices_info()
         known_indices = self._get_all_known_indices_name()
-        for index_name in existing_indices.keys():
+        for index_name in sorted(existing_indices.keys()):
```

Comment:
Just curious why we want to sort indices? Is it for the readability of the log?

Reply:
This was for the tests, while mocking the `builtins.input` it was essential that we have particular order in which we provide the index names.